subworkflows/local/bigstitcher_spark/tests/full-stitching-pipeline.nf

```diff
@@ -0,0 +1,130 @@
+include { BIGSTITCHER_SPARK as STITCH           } from '../main'
+include { BIGSTITCHER_SPARK as CREATE_CONTAINER } from '../main'
+include { BIGSTITCHER_SPARK as FUSE             } from '../main'
+
+workflow FULL_STITCHING_WORKFLOW {
+    take:
+    ch_input                // channel: [ val(meta), path(stitching_data) ]
+    spark_config            // map: Additional spark config properties
+    distributed_cluster     // boolean: use a distributed cluster
+    work_dir                // string | file: working directory
+    spark_workers           // int: number of workers in the cluster
+    min_spark_workers       // int: minimum required workers
+    spark_worker_cpus       // int: number of CPUs per worker
+    spark_mem_gb_per_cpu    // int: memory in GB per worker core
+    spark_driver_cpus       // int: number of CPUs for the driver
+    spark_driver_mem_gb     // int: driver memory in GB
+
+    main:
+    def p1 = spark_config
+    def p2 = distributed_cluster
+    def p3 = work_dir
+    def p4 = spark_workers
+    def p5 = min_spark_workers
+    def p6 = spark_worker_cpus
+    def p7 = spark_mem_gb_per_cpu
+    def p8 = spark_driver_cpus
+    def p9 = spark_driver_mem_gb
```

Comment:
nextflow complains that "variable is already used" if I don't assign it to a local variable